src/main/javahelp/org/zaproxy/zap/extension/hud/resources/help/contents/options.html

```diff
@@ -17,6 +17,19 @@ <h2>Enable when using ZAP in daemon mode</h2>
 	This defaults to false as many people use ZAP for automation, injecting the HUD would not help and could cause
 	existing tests to break.
 
+	<h2>Enable anonymous telemetry</h2>
+	
+	When set will send pings to a bit.ly URL when:
+	<UL>
+	<LI>The HUD is used
```

Comment:
This does not seem to be the case, yet? (`hud_start` is not used?)

Reply:
Ugh - 'tidied' up too much of my code locally :P